std/maybe.sy

```diff
@@ -0,0 +1,30 @@
+
+Maybe :: enum
+    Just *
+    None
+end
+
+fromMaybe :: fn maybe, a ->
```

Comment:
`maybe.orDefault` feels like a more descriptive name for this function.

Reply:
I see now that this is stolen from Haskell, while my suggestion is from Rust. I think the Rust name is better though :angel:

Reply:
I think it is a lot more intuitive with or as well.

Reply:
The Haskell function has the added context of saying what type you give in. I don't personally agree but I am being easily swayed today.